src/debian/01_password

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/sh
+
+GRUB_PASS=
```

Comment:
Empty?

Reply:
the GRUB_PASS is modified with ansible to permit usage of a ansible var

Reply:
Use jinja to explicit that this is a template file. This is more ansible-y than runing sed/awk commands.